### PR TITLE
eo-cd changes

### PR DESCRIPTION
**Pull request by CodingDegen**

### CODE REVIEW

File: .github/workflows/eo-cd.yml [hunk 1]
```diff
@@ -104,14 +104,11 @@ jobs:
     needs: affected
     runs-on: ubuntu-latest
     outputs:
-      version: ${{ steps.version.outputs.semver }}
+      version: ${{ steps.version.outputs }}
```
Comment (CodeReaper) [inline .github/workflows/eo-cd.yml:107]: You are missing the output name here.

File: .github/workflows/eo-cd.yml [hunk 2]
```diff
@@ -104,14 +104,11 @@ jobs:
...
     steps:
       - name: Get version
         id: version
-        uses: Energinet-DataHub/energy-origin/.github/actions/yaml-semver@main
-        with:
-          yaml-file: build/infrastructure/eo/configuration.yaml
-          yaml-path: version
+        uses: Energinet-DataHub/energy-origin/.github/actions/docker-image-version
```
Comment (CodeReaper) [inline .github/workflows/eo-cd.yml:111]: This should be constricted to main

File: .github/workflows/eo-cd.yml [hunk 3]
```diff
@@ -165,19 +162,3 @@ jobs:
...
-    runs-on: ubuntu-latest
-    name: Bump configuration version
-    if: ${{ github.event_name == 'pull_request' && needs.affected.outputs.is-affected == 'true' }}
-    needs:
-      - affected
-      - update-environment
-    steps:
-      - uses: actions/checkout@v3
-
-      - uses: ./.github/actions/yaml-increment-version
```
Comment (dzhavat) [inline .github/workflows/eo-cd.yml:179]: suggestion: as part of this PR, can you also delete the `.github/actions/yaml-increment-version` workflow? Its only purpose is to [automatically] bump the version, which will not be needed anymore.

praise: great job getting rid of the version bump 💯

Comment (CodingDegen) [reply .github/workflows/eo-cd.yml:179]: Can you verify that it's not being used anywhere else than the eo? We were not sure, but will gladly remove it, if its not being used anywhere.

Comment (dzhavat) [reply .github/workflows/eo-cd.yml:179]: Yes, it's not used anywhere else. It can be removed.

Comment (CodingDegen) [reply .github/workflows/eo-cd.yml:179]: Awesome, thanks for the suggestion.

**Review by CodeReaper — APPROVED** [review]

**Pull request merged**